src/pages/GroupDetails/GroupDetails.js

```diff
@@ -34,6 +34,12 @@ const GroupDetails = ({ groupId }) => {
 
     return (
         <Details title={group.displayName}>
+            {group.access.read && (
```

Comment:
Hi @Chisomchima. I think you need to display this when `!group?.access?.write`.

At the moment, because you display the message whenever a user has `read` access, the user will see this message if they have both `read` and `write` access to the group, e.g.:

<img width="700" alt="image" src="https://github.com/dhis2/user-app/assets/18490902/1363b0c2-7281-4a8a-864c-5df4c8cc68b1">